CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Unreleased
+
+* Freezes all string literals in the library - possibly breaking apps that edit them
```

Comment:
Maybe something like:

> Freeze all string literals. This should have no impact unless your application is modifying ('monkeypatching') the internals of the library in a unusual way.

Reply:
Yeah that's great! I wonder if this warrants a major version bump for the next release.

Reply:
We're not changing the public interface, but perhaps out of caution we should have a major bump.